lib/std/zig/tokenizer.zig

```diff
@@ -1001,6 +1000,17 @@ pub const Tokenizer = struct {
                     },
                 },
 
+                .period_asterisk => switch (c) {
+                    '*' => {
+                        result.id = .Invalid;
```

Comment:
To get a proper error message you have to give this a new tag. You should be able to copy what I did with `.Invalid_ampersands`.